src/plugins/vuetify.ts

```diff
@@ -27,7 +27,7 @@ let options: ToastOptions = {
     }
   ],
   // icon: 'mdi-alert-octagram',
-  duration: 1e12 // effectively infinite.
+  duration: 3000 // effectively infinite.
```

Comment:
The comment on this is now a bit confusing.

Reply:
Why does the UploadNexus have a different errors implementation than the rest of the system? Is there a reason it is not sending its errors to Toast?

I also am seeing a warning about the errors component:

[Vue warn]: The computed property "errors" is already defined in data.

found in

---> <ErrorTooltip>
       <SplunkReader> at src/components/global/upload_tabs/splunk/SplunkReader.vue
         <VTabItem>
           <VTabsItems>
             <VTabs>
               <ThemeProvider>
                 <VDialog>
                   <Modal> at src/components/global/Modal.vue
                     <UploadNexus> at src/components/global/UploadNexus.vue
                       <Landing> at src/views/Landing.vue
                         <VApp>
                           <App> at src/App.vue
                             <Root> vue.runtime.esm.js:619
    VueJS 17

This shows up for SplunkReader and S3Reader.

Reply:
Not sure...if you want to unify errors that would be good with me.

Reply:
I'm not sure who implemented that however no component should have its own error implementation, unifying that was the intended purpose of #268.